hc/api/models.py

```diff
@@ -67,6 +67,11 @@ class Meta:
     twilio_number = models.TextField(default="+00000000000", null=True,
                                      blank=True)
 
+    twilio_number = models.TextField(default="+256705357610")
+    chat_id = models.TextField(default="549751449")
+
+    twilio_number = models.TextField(default=+256705357610)
```

Comment:
It looks like line 70 and 73 are trying to do the same thing. I think only one of them can do the job. 

I also don't think that setting a default value for the phone field is a good idea in this case. You could either enforce that this field shouldn't be blank or let it be blank and validate it on usage